docs/cli-reference/dfx-cycles.md

```diff
@@ -73,41 +73,81 @@ Use the `dfx cycles transfer` command to transfer cycles from your account to an
 ### Basic usage
 
 ``` bash
-dfx cycles transfer [options] <amount>
+dfx cycles transfer [options] <to> <amount>
 ```
 
 ### Arguments
 
-You must specify the following argument for the `dfx cycles transfer` command.
+You must specify the following arguments for the `dfx cycles transfer` command.
 
-| Argument     | Description                       |
-|--------------|-----------------------------------|
-| `<amount>`   | The number of cycles to transfer. |
+| Argument   | Description                       |
+|------------|-----------------------------------|
+| `<to>`     | The principal of the account to which you want to transfer cycles. |
+| `<amount>` | The number of cycles to transfer. |
 
 ### Options
 
 You can specify the following options for the `dfx cycles transfer` command.
 
-| Option                           | Description                                                        |
-|----------------------------------|--------------------------------------------------------------------|
-| `--to-owner <principal>`         | The principal of the account to which you want to transfer cycles. |
-| `--to-subaccount <subaccount>`   | The subaccount to which you want to transfer cycles.               |
-| `--from-subaccount <subaccount>` | The subaccount from which you want to transfer cycles.             |
-| `--fee <fee>`                    | Specifies a transaction fee.                                       |
-| `--memo <memo>`                  | Specifies a numeric memo for this transaction.                     |
+| Option                           | Description                                                                            |
+|----------------------------------|----------------------------------------------------------------------------------------|
+| `--to-subaccount <subaccount>`   | The subaccount to which you want to transfer cycles.                                   |
+| `--from-subaccount <subaccount>` | The subaccount from which you want to transfer cycles.                                 |
+| `--memo <memo>`                  | Specifies a numeric memo for this transaction. Can only be passed in `--to-owner mode. |
 | `--created-at-time <timestamp>`  | Specify the timestamp-nanoseconds for the `created_at_time` field on the transfer request. Useful for controlling transaction-de-duplication. https://internetcomputer.org/docs/current/developer-docs/integrations/icrc-1/#transaction-deduplication- |
 
 ### Examples
 
 Transfer 1 billion cycles to another account:
 
 ``` bash
-dfx cycles transfer 1000000000 --to-owner raxcz-bidhr-evrzj-qyivt-nht5a-eltcc-24qfc-o6cvi-hfw7j-dcecz-kae --network ic
+dfx cycles transfer raxcz-bidhr-evrzj-qyivt-nht5a-eltcc-24qfc-o6cvi-hfw7j-dcecz-kae 1000000000 --network ic
```

Comment:
Is there going to be support to specify the amount in a more human-readable format like `100B` or `10T`?

Reply:
That's a great idea for a future improvement.  I'll create a ticket.